wiki/Welcome/de.md

```diff
@@ -1,130 +1,117 @@
+### Schritt 2 - Es herunterladen!
 
-<img src="welcome01.png" title="welcome01.png" alt="welcome01.png" width="580" />
+Wenn du nach einer freigegebenen Beatmap eines Liedes Ausschau halten solltest, werden dir alle vorhanden angezeigt. Wie auch immer, sei nicht sauer, wenn ein Lied nicht mit dabei sein sollte. Es sind jede Menge Unranked Beatmaps vorhanden, die derzeit noch in Bearbeitung sind, worunter auch dein Lieblingslied dabei sein könnte. Bitte **erstelle keine Anfragen für Beatmaps, die du gerne hättest**, die meisten Mapper werden das Mappen, was **sie** mögen und nicht was du magst. Dies könnte jedoch ein perfekter Start für dich sein, um deine Karriere als Mapper zu starten!
 
-Du findest mindestens einmal einen großen "Download Beatmap" Knopf auf jeder Seite der Beatmaps (Du musst mit deinem Account angemeldet sein, ansonsten wird dir ein gelber Knopf zur Anmeldung angezeigt).
+![Beatmap Auflistung Suchergebnisse](Welcome01.png "Beatmap Auflistung Suchergebnisse")
 
-<img src="DE-Welcome02.jpg" title="DE-Welcome02.jpg" alt="DE-Welcome02.jpg" width="580" />
+Du findest mindestens einmal einen großen `Download Beatmap` Knopf auf jeder Seite der Beatmaps (du musst mit deinem Account angemeldet sein, ansonsten wird dir ein gelber Knopf zur Anmeldung angezeigt).
 
-**Achtung:** Es spielt keine Rolle, wo die Datei gespeichert wird, solange du die Beatmap nach dem Download noch findest. Darüber hinaus kannst du die Datei direkt öffnen lassen, wodurch osu! zum Starten gezwungen wird, wenn es nicht eingeschaltet sein sollte.
+![Beatmap runterladen](Welcome02.png "Beatmap runterladen")
 
-Schritt 3 - Lade es auf osu!
-----------------------------
+**Achtung:** Es spielt keine Rolle, wo die Datei gespeichert wird, solange du die Beatmap nach dem Download noch findest. Darüber hinaus kannst du die Datei direkt öffnen lassen, wodurch *osu!* zum Starten gezwungen wird, wenn es nicht eingeschaltet sein sollte.
 
-![".osz" Datei, einwandfreie Beatmap-Datei](Osz icon.png "fig:".osz" Datei, einwandfreie Beatmap-Datei") Nachdem du die Beatmap fertig heruntergeladen haben, solltest du ohne Weiteres in der Lage sein die **.osz** Datei zu öffnen. Wenn es dir aus einigen Gründen nicht gefallen sollte, können Sie die Datei auch einfach auf osu! ziehen.
+### Schritt 3 - Lade es auf osu!
 
-<img src="DE-Welcome03.jpg" title="DE-Welcome03.jpg" alt="DE-Welcome03.jpg" width="580" />
+![".osz" Datei, einwandfreie Beatmap-Datei](Osz icon.png "fig:".osz" Datei, einwandfreie Beatmap-Datei") Nachdem du die Beatmap fertig heruntergeladen hast, solltest du ohne Weiteres in der Lage sein die **.osz** Datei zu öffnen. Zum Öffnen der Datei klick entweder darauf oder zieh die Datei in *osu!* hinein.
```

Comment:
I don't know what happened here but the image is broken

also, `**.osz**` -> `` `.osz` `` (use code)